static/js/lib/page.js

```diff
@@ -0,0 +1,20 @@
+// Evaluates code only on a page defined by the given selector.
+//
+// If the selector is not specific enough and more than one container matches,
+// an error is shown.
+//
+// The callback is invoked with a jQuery object wrapping the selected
+// container.
+//
+function Page(selector, callback) {
```

Comment:
This is a simple wrapper around a common pattern for page selection. I'll apply it to all "page" javascripts in a later PR.